test/ssc_test/cmod_battery_test.cpp

```diff
@@ -186,3 +186,55 @@ TEST_F(CMBattery_cmod_battery, ResilienceMetricsFullLoadLifetime){
     }
     EXPECT_EQ(max_indices[0], 2);
 }
+
+/// Test standalone battery compute modeule with a input lifetime generation and merchant plant prices
+TEST_F(CMBattery_cmod_battery_fom, MerchantPlantLifetimeAutomatic) {
```

Comment:
Was lifetime model crucial to the case being fixed here?

Reply:
Good call. I converted the test to a single year.